.gitignore

```diff
@@ -26,12 +24,10 @@ chsdi/static/doc/build/
 chsdi/static/css/extended.min.css
 chsdi/static/css/blueimp-gallery.min.css
 chsdi/static/info.json
-deploy/hooks/pre-create-code
 node_modules
 MANIFEST
 buildout_branch.cfg
```

Comment:
Same for this ?

Reply:
Good catches, thanks! I'll scan through the .gitignore again and try to find everything that I did not find in the first shot